Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM php:5.6
+
+RUN echo "date.timezone = UTC" > /usr/local/etc/php/conf.d/date.ini
+
+VOLUME /app
+WORKDIR /app
+
+ENTRYPOINT ["./app/console"]
+CMD ["server:run", "0.0.0.0:8000"]
```

Comment:
I vote for `8000` port because it used as a default port for `app/console server:run` command.